Makefile

```diff
@@ -436,22 +436,22 @@ endif
 
 # The rootfs partition size is set to 512MB after 10.2.0 release (see commit 719b4d516)
 # Before 10.2.0 it was 300MB. We must maintain compatibility with older versions so rootfs size cannot exceed 300MB.
-# kubevirt and nvidia are not affected by this limitation because there no installation of kubevirt prior to 10.2.0
-# Nevertheless lets check for ROOTFS_MAXSIZE_MB not exceeding 900MB for kubevirt, 450MB for NVIDIA based platforms (arm64) and 270MB for x86_64 and other arm64 platforms
+# 'k' and nvidia are not affected by this limitation because there no installation of kubevirt/k3s prior to 10.2.0
+# Nevertheless lets check for ROOTFS_MAXSIZE_MB not exceeding 900MB for 'k', 450MB for NVIDIA based platforms (arm64) and 270MB for x86_64 and other arm64 platforms
 # That helps in catching image size increases earlier than at later stage.
 # We are currently filtering out a few packages from bulk builds since they are not getting published in Docker HUB
-ifeq ($(HV),kubevirt)
+ifeq ($(HV),k)
         PKGS_$(ZARCH)=$(shell find pkg -maxdepth 1 -type d | grep -Ev "eve|alpine|sources$$")
         ROOTFS_MAXSIZE_MB=900
 else
-        #kube container will not be in non-kubevirt builds
+        #kube container will not be in non-k= builds
         PKGS_$(ZARCH)=$(shell find pkg -maxdepth 1 -type d | grep -Ev "eve|alpine|sources|kube|external-boot-image$$")
         # evaluation platform is not limited by rootfs size, set to some large value
         ifeq ($(PLATFORM),evaluation)
             ROOTFS_MAXSIZE_MB=9999
         # nvidia platform requires more space
         else ifeq (, $(findstring nvidia,$(PLATFORM)))
-            ROOTFS_MAXSIZE_MB=280
+            ROOTFS_MAXSIZE_MB=285
```

Comment:
Why is this value bumped up ?

Reply:
See test in description (updating the cross-pkg vendoring in eve to get the new IsHyType somehow grew all images by 7 MB! I think this had been forgotten for a while.